src/Resources/translations/CmfSonataAdminIntegrationBundle.de.xliff

```diff
@@ -287,8 +284,7 @@
             <!-- $Core -->
             <trans-unit id="form.tab_publish">
                 <source>form.tab_publish</source>
-                <!-- TODO: Translate -->
-                <target>Publish Settings</target>
+                <target>Einstellungen zum Veröffentlichen</target>
```

Comment:
Fine by me. I kinda used the [Drupal Glossary](https://localize.drupal.org/node/1263) when translating into Dutch and they use "veröffentlichen" as well, so let's stick with that!